assets/css/v2/dup/free_text.scss

```diff
@@ -150,11 +152,11 @@
   }
 
   .free-text__route-pill {
-    height: 144px;
+    height: 128px;
     width: 248px;
     border-radius: 72px;
     font-size: 96px;
-    line-height: 144px;
+    line-height: 117.22px;
```

Comment:
This is oddly precise—any reason why it can't be a whole number, at least?

Same comment for the other decimal-valued line-heights below in this file.

Reply:
O wow, I didn't even notice how precise the decimal is. I honestly just pulled that number from the designs. I think the weird decimal may have been cleaned up though, as the number is just 117 now. I'll change this to match.